docs/en/writing-running-appium/tutorial/swipe/ios-mobile-screen.md

```diff
@@ -0,0 +1,92 @@
+## iOS 'mobile:': Screen swipe
+
+XCTest framework supports unique gestures like "mobile:scroll" and
+"mobile:swipe". They are not so flexible as UIScrollable but still
+useful.
+
+As usual swipe performs swipe action while scroll tries to change one
```

Comment:
This sentence does not make sense to me

Reply:
well. normally beginners do not understand difference between swipe and scroll. we need to mention this. suggest other text.

Reply:
for me personally these both are just finger moves with different timings

Reply:
no. swipe just finger moves. scroll not. scroll tries to change exactly ONE page.